fastcrypto-zkp/src/api.rs

```diff
@@ -31,8 +33,13 @@ pub fn verify_groth16_in_bytes(
     proof_public_inputs_as_bytes: &[u8],
     proof_points_as_bytes: &[u8],
 ) -> Result<bool, FastCryptoError> {
-    let x = BlsFr::deserialize_compressed(proof_public_inputs_as_bytes)
-        .map_err(|_| FastCryptoError::InvalidInput)?;
+    let mut x = Vec::new();
+    for chunk in proof_public_inputs_as_bytes.chunks(SCALAR_LENGTH) {
```

Comment:
Can we document that we get a concatenated bytes, then split the bytes into SCALAR_LENGTH chuncks?